src/jgikbase/idmapping/cli.py

```diff
@@ -70,16 +68,8 @@ def execute(self) -> int:
             return 1
         if a.list_users:
             return self._list_users(luh, a.verbose)
-        # ok, so user must have a value at this point
-        if sum((a.create, bool(a.admin), a.new_token)) != 1:
-            self._stderr.write('Exactly one of {}, {}, or {} must be specified.\n'.format(
-                self._CREATE, self._NEW_TOKEN, self._ADMIN))
-            return 1
-        try:
-            u = Username(a.user)
-        except Exception as e:
-            self._handle_error(e, a.verbose)
-            return 1
+        # ok, so user must have a valid value at this point
+        u = Username(a.user)
```

Comment:
It's a little confusing why you make a call to Username() here, but also make the same call in _check_inputs below. Maybe that call can be removed from _check_inputs and any exceptions from that call here can be handled with the decorator in the future

Reply:
I check the username in check inputs so the CLI can fail fast if the username is invalid. Otherwise the CLI will wait for the db to start and *then* check the username and fail.

The 2nd call is because the functions that will be called later expect a username.